cmd/cluster/template/template.go

```diff
@@ -67,6 +66,20 @@ func RunCmd(cmd *cobra.Command) error {
 		cc.OsTag = cc.KubeVersion
 	}
 
+	// if cluster name is empty, then default it to ocne
+	if cc.Name == "" {
+		cc.Name = "ocne"
+	}
+
+	// if number of control plane nodes is 0, then default it to 1
+	if cc.ControlPlaneNodes == 0 {
+		cc.ControlPlaneNodes = pkgconst.ControlPlaneNodes
+	}
+
+	if cc.Provider == "oci" && cc.ControlPlaneNodes % 2 == 0 {
```

Comment:
When setting the number of control plane nodes to an even number for capi cluster creation, I received below error:
```shell
ERRO[2024-10-01T01:58:05Z] admission webhook "validation.kubeadmcontrolplane.controlplane.cluster.x-k8s.io" denied the request: KubeadmControlPlane.cluster.x-k8s.io "pras-cluster-control-plane" is invalid: spec.replicas: Forbidden: cannot be an even number when etcd is stacked
```

So added validation that the number of control plane nodes is odd for quorum purposes.